src/Compatibility/Core/src/iOS/Forms.cs

```diff
@@ -33,6 +31,11 @@ public static class Forms
 	{
 		public static bool IsInitialized { get; private set; }
 
+		static IFontManager s_fontManager;
+
+		internal static IFontManager FontManager =>
```

Comment:
For the time being, instead of storing the fonts in a static dictionary in `FontRegistrar` I split the functionality into the `FontRegistrar` that is the part that manages a list of fonts and `FontManager` that manages the font lifetime. Both are instances and can be obtained via the service provider in Core. However for forms, everything expects a static object. So to fix that, I added a static property to Forms, which is just for compatibility.

This might be wrong as the provider will return a different instance to this property as it is new-ed up here. Not sure what the correct way to do this is.